armi/utils/mathematics.py

```diff
@@ -515,7 +516,7 @@ def parabolaFromPoints(p1, p2, p3):
         print("Error in parabola {} {}".format(A, b))
         raise
 
-    return float(x[0]), float(x[1]), float(x[2])
+    return float(x[0][0]), float(x[1][0]), float(x[2][0])
```

Comment:
```suggestion
    # x returns `[num]` instead of `num`, so we have to grab the first/only element in that list
    return float(x[0][0]), float(x[1][0]), float(x[2][0])
```